lib/features/component/editText/username_editText.dart

```diff
@@ -43,3 +46,46 @@ class UsernameEditText extends StatelessWidget {
     );
   }
 }
+
+class FullNameEditText extends StatelessWidget {
```

Comment:
change name to be more generic, as this is not only going to be used for fullname